chain/network/src/peer_manager/network_state/routing.rs

```diff
@@ -90,10 +90,7 @@ impl NetworkState {
         // instead, however that would be backward incompatible, so it can be introduced in
         // PROTOCOL_VERSION 60 earliest.
         let (edges, ok) = self.graph.verify(edges).await;
-        let result = match ok {
-            true => Ok(()),
-            false => Err(ReasonForBan::InvalidEdge),
-        };
+        let result = if ok { Ok(()) } else { Err(ReasonForBan::InvalidEdge) };
```

Comment:
Can the `result` computation be moved into the `if` block below?  It seems like we are using `result` only there.  Of course there might be some ownership issues.